tests/e2e-driver/internal/runner/config.go

```diff
@@ -0,0 +1,33 @@
+package runner
+
+import (
+	"fmt"
+	"io/ioutil"
+
+	"gopkg.in/yaml.v3"
+)
+
+// TestConfig is the config for a test
+type TestConfig struct {
+	PkgPath string `json:"pkgPath,omitempty" yaml:"pkgPath,omitempty"`
+	Network bool   `json:"network,omitempty" yaml:"network,omitempty"`
+}
+
+// TestConfigs contains a list of TestConfig
+type TestConfigs struct {
+	Configs []TestConfig `json:"configs,omitempty" yaml:"configs,omitempty"`
+}
+
+// ConfigFromFile returns a list of TestConfig read from path
+func ConfigFromFile(path string) (*TestConfigs, error) {
```

Comment:
Where's the actual config file passed to this?

Reply:
Sorry I don't get your point. Can you clarify?